contentcuration/contentcuration/frontend/channelEdit/views/CurrentTopicView.vue

```diff
@@ -1016,4 +1016,8 @@
     flex-shrink: 0;
   }
 
+  .text-truncate{
+    overflow: hidden;
+    width:100px;
```

Comment:
I think this would not only partially resolve the issue but would also introduce responsiveness concerns. Besides, if we really need to override the width of the component, it makes more sense do so within a wrapper instead, I think (this styling has been defined within the component itself). Better still, we can make the change in the Breadcrumbs component where I suspect the issues resides. It appears to me that the finickiness is due to the way we are handling the overflow calculations, so it will require some digging around to establish what is causing the issue.
https://github.com/learningequality/studio/blob/4dc29f235e2667829dfc618e05cd26c2b79b5b4d/contentcuration/contentcuration/frontend/shared/views/Breadcrumbs.vue#L95-L111